src/lib/swapper/manager/SwapperManager.test.ts

```diff
@@ -136,69 +150,53 @@ describe('SwapperManager', () => {
 
   describe('getSupportedBuyAssetIdsFromSellId', () => {
     it('should return an array of supported buy assetIds given a sell asset Id', () => {
-      const assetIds = [
-        'bip122:000000000019d6689c085ae165831e93/slip44:0',
-        'eip155:1/erc20:0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9', // Aave
-        'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d', // Fox
-      ]
+      const assetIds = [BTC.assetId, WETH.assetId, FOX.assetId]
 
-      const sellAssetId = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
+      const sellAssetId = FOX.assetId
       const swapperManager = new SwapperManager()
       swapperManager.addSwapper(zrxEthereumSwapper)
 
       expect(
         swapperManager.getSupportedBuyAssetIdsFromSellId({ sellAssetId, assetIds }),
-      ).toStrictEqual(assetIds.slice(-2))
+      ).toStrictEqual([WETH.assetId, FOX.assetId])
     })
 
     it('should return unique assetIds', () => {
-      const assetIds = [
-        'bip122:000000000019d6689c085ae165831e93/slip44:0',
-        'eip155:1/erc20:0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9', // Aave
-        'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d', // Fox
-        'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d', // Fox (duplicate)
-      ]
+      const assetIds = [BTC.assetId, WETH.assetId, FOX.assetId, FOX.assetId]
 
       const sellAssetId = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
```

Comment:
While at it, can we use BTC or WETH as a duplicate instead and remove FOX.assetId from here? FOX can't be buyAsset for FOX sellAsset
```suggestion
      const sellAssetId = foxAssetId
```